mobile/app/lib/nativescript-ws.ts

```diff
@@ -29,7 +29,7 @@ export class NativescriptWebSocketAdapter extends WebSocketAdapter {
 
     /** @inheritdoc */
     onMessage(callback: (data: Buffer) => void): void {
-        this.ws.on("message", (socket, msg) => {
+        this.ws.on("message", (socket, msg: ArrayBuffer) => {
             if (msg instanceof Buffer || msg instanceof ArrayBuffer) {
```

Comment:
as `msg` is an ArrayBuffer, it will always be true.

Reply:
I'm not sure that `nativescript-websockets` always gives an `ArrayBuffer` here - I'll remove the method signature and keep the test...